### DISCUSSION TITLE
Permissions to Add Card

### DISCUSSION THREAD

**Discussion by TechieBlossom** [ibhavikmakwana/supabase_playground#17] [Q&A] [answered]
Hi, @ibhavikmakwana This is nice !!
I am looking to contribute to this project and want to create a card. Please give permission to allow creating card and raising MRs.

**Comment by ibhavikmakwana** [comment, marked as answer]
Hey @TechieBlossom Would you mind creating an issue for the same? I'll create a card for it. I am thinking of giving collaboration access to the active contributors only.
